MidCamp/midcamp
# PHP 7.4

# Description

- Updates environments to use PHP 7.4
- Updates [Name](https://www.drupal.org/project/name) module to version 1.0-rc3 for PHP 7.4 compliance
- Updates [Paragraphs](https://www.drupal.org/project/paragraphs) module to version 1.12 for PHP 7.4 compliance
- Updates [Devel](https://www.drupal.org/project/devel) module to version 2.1 for PHP 7.4 compliance

# Test instructions

- Run `lando rebuild -y` to rebuild local containers
- Run `lando php -v` and confirm output like:
    ```shell
    PHP 7.4.22 (cli) (built: Jul 30 2021 02:07:33) ( NTS )
    Copyright (c) The PHP Group
    Zend Engine v3.4.0, Copyright (c) Zend Technologies
        with Zend OPcache v7.4.22, Copyright (c), by Zend Technologies
    ```
- Visit the [Status report page](http://midcamp-org.lndo.site/admin/reports/status) and confirm PHP 7.4 is reported
- Run `lando drush updb -y` to confirm there are no pending database updates
- Test the Name field (on the People entity) and ensure it still functions
- Test Paragraphs (on the Article node type) and ensure they still function
